gr-qtgui/grc/qtgui_freq_sink_x.block.yml

```diff
@@ -20,8 +20,9 @@ parameters:
     hide: ${ ('none' if len(name) > 0 else 'part') }
 -   id: fftsize
     label: FFT Size
-    dtype: int
+    dtype: enum
```

Comment:
If you keep these as int, the dropdown persists, but the value can also be set to a variable